src/main.cpp

```diff
@@ -190,13 +190,15 @@ char uart6slave_line[32];
 static volatile uint32_t minda_falling_edges = 0;
 uint32_t get_Z_probe_endstop_hits() { return minda_falling_edges; }
 
+static uint8_t is_flashing = 0;
```

Comment:
I assume this is a very brutal piece of development code ... is_flashing variable should be probably hidden somewhere in a more appropriate place related to the code being used in.